src/Tizen.Uix.Tts/Tizen.Uix.Tts/TtsClient.cs

```diff
@@ -673,6 +752,32 @@ public State CurrentState
 
         }
 
+        /// <summary>
+        /// Gets the current state of TTS service.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        /// <value>
+        /// The current state of TTS service.
+        /// </value>
+        /// <returns>
+        /// Current TTS service state value.
+        /// </returns>
```

Comment:
\<returns\> for a property is ignored as not related to this element, you can remove those 3 lines

Reply:
Thank you for checking. I will remove it soon.